public/app/features/alerting/unified/home/Home.tsx

```diff
@@ -56,7 +58,7 @@ export function getHomeApp(insightsEnabled: boolean) {
 }
 
 export default function Home() {
-  const insightsEnabled = !!config.featureToggles.alertingInsights;
+  const insightsEnabled = (!isOpenSourceEdition() || isDevelopment()) && !!config.featureToggles.alertingInsights;
```

Comment:
Do we want to allow OSS users in development mode to be able to use this feature toggle?. If so, I would update the PR title and description. wdyt?

Reply:
I believe the mode `development` is only used for local environments so it seemed safe to use here. I copied this from the Declare Incident button implementation. @gillesdemey could you confirm if the scenario that Sonia mentioned is possible?